library/std/primitive_docs/string_string.md

```diff
@@ -0,0 +1 @@
+string::String
```

Comment:
Why not just use the raw HTML links for `std` as well?

Reply:
Switching to raw links breaks when the primitives are re-exported; take a look at all the new linkcheck exceptions I added for core. If we used raw links for std, they would be broken for std too.